### PR TITLE
Hotfix/fix pagination at end of table #161465449

### PR DESCRIPTION
**Pull request by chrisdolendo**
…irst time

### CODE REVIEW

**Review by akegan — APPROVED** [review]

**Comment by akegan** [comment]
@fddayan this won't get merged/deployed until tomorrow morning most likely, so take a look and let us know if you have any concerns!

File: app/services/force/soql_query_builder.rb [hunk 1]
```diff
@@ -73,7 +73,7 @@ def page_size
     end
 
     def pages
-      total_size / page_size
+      total_size.to_f / page_size.to_f
```
Comment (fddayan) [inline app/services/force/soql_query_builder.rb:76]: 👍 good catch

**Review by fddayan — APPROVED** [review]
Looks good 👍

**Comment by chrisdolendo** [comment]
I can’t review this since I created the PR but I 👍🏽 @akegan’s latest commit 

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Pull request merged**